src/main/java/jenkins/plugins/slack/workflow/SlackResponse.java

```diff
@@ -0,0 +1,39 @@
+package jenkins.plugins.slack.workflow;
+
+import org.apache.commons.lang.StringUtils;
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.Whitelisted;
+import org.json.JSONObject;
+
+import java.io.Serializable;
+
+public class SlackResponse implements Serializable {
+    private String channelId;
+    private String ts;
+
+    public SlackResponse(String slackResponseString) {
+        if (!StringUtils.isEmpty(slackResponseString)) {
```

Comment:
won't this be `ok` for a non bot user?

Reply:
currently it only registers the response for a bot user, so this should be `null` for a non bot user. This is a precaution against any other case that I might not be aware of. I can change it to check for `ok` explicitly, I just wasn't sure if that would cover everything